gulpfile.js

```diff
@@ -223,4 +232,9 @@ gulp.task('watch', function() {
 // This task starts google app engine development server.
 // TODO(Barnabas Makonda): check if files are already generated and if so
 // do not build.
-gulp.task('start_devserver', ['build', 'gulpStartGae', 'watch']);
+var isProdMode = argv.prod_env === 'True';
+if (isProdMode) {
+  gulp.task('start_devserver', ['gulpStartGae', 'watch']);
```

Comment:
I'd say, do the thing that results in the least convoluted build system possible. If this involves moving everything to gulp, then that's what needs to be done.